GameBoard/Startup.cs

```diff
@@ -29,12 +31,21 @@ public void ConfigureServices(IServiceCollection services)
                     options.MinimumSameSitePolicy = SameSiteMode.None;
                 });
 
-            LogicLayer.Configuration.ConfigureDbContext(
-                services,
-                Configuration.GetConnectionString("DefaultConnection"));
+            if (Environment.IsDevelopment())
```

Comment:
This `if` can be shortened using `Environment.IsDevelopment() ? "Gameboard_Develop" : "Gameboard_Release"`.